### ISSUE TITLE
Image configuration issue

### ISSUE THREAD

**Issue by donglin1111** [RoboCup-SSL/ssl-vision#91]
![webwxgetmsgimg 1](https://user-images.githubusercontent.com/36018956/37439899-99f40652-27d0-11e8-8468-34cad284a224.jpg)

We got some issues shown as above that the camera can detecte the robot color, but it can not recognize the robot ID. I am not sure if the team configuration is correct or not. 
Could you give me some advices?
Thank You.

**Comment by g3force** [comment]
The team config is wrong. There are several parameters that look strange. I suggest you compare it with the default values. You would, for example, check out the repo into a new folder and start ssl-vision from there. This way, the default config will be generated.

Usually, the only thing you have to change in the team config is the robot height.

**Comment by donglin1111** [comment]
![screenshot from 2018-03-16 13-25-05](https://user-images.githubusercontent.com/36018956/37535346-d4b308ee-291d-11e8-801e-25160cb64840.png)

I reset the ssl-vision, but It still has same issue. What does the robot height's mean? 
Could you give me more suggests?

**Comment by g3force** [comment]
The team config is still wrong... Try resetting the file:
`git checkout robocup-ssl-teams.xml`

A fresh checkout looks like this for me:
![image](https://user-images.githubusercontent.com/779094/37537080-c049cb84-294c-11e8-9cb6-ac349e402904.png)

The robot height should ideally be the actual height of your robots.

**Comment by donglin1111** [comment]
I checked our documentation that the robot height it is 145mm. 
However, once we select the Unique pattern, the robot doesn't show up on the graphicalClient.

**Comment by g3force** [comment]
can you show me the detection with threshold and blobs? And with geometry calibration? How high is your camera attached?

**Comment by donglin1111** [comment]
![screenshot from 2018-03-19 15-19-30](https://user-images.githubusercontent.com/36018956/37617288-1829751e-2b89-11e8-88b3-b382004ffd5b.png)

Here are our detection with threshold and blobs, also the geometry calibration of half field. Now we have only the ball showing in the graphical client, no robot.

**Comment by g3force** [comment]
are you sure that the geometry calibration is correct? It looks like you use the quad size field sizes with a smaller field.
There are several filters that use the geometry to filter out unrealistic blobs, so the geometry must be reasonable.

**Comment by donglin1111** [comment]
seems like that the documentation is not quite specific on the field configuration details. In the SSL vision UI, all the parameters under field configuration have no units, the default values are 12000 for field length, 9000 for field width......and so on. I tried to find the units of these numbers but I just did not see it. Also I believe that we need to tell the SSL vision that how many cameras we are actually using before we do the field configuration, because currently we need to put the four bullet points (the four blue points) at the quad positions of the whole field to make the configuration looks like that it is correct......

**Comment by g3force** [comment]
The default values are for division A (quad size field). Isn't it obvious that the unit is mm? Maybe, we could add it to the label.
With the latest master, you should be able to configure the number of cameras. This is a new feature. If you miss some more details, please tell me and I will update the documentation.

**Comment by donglin1111** [comment]
When you say "quad size", you mean that is the size of 1/4 of the whole filed? I'm sorry about my language issues here......
Another issue is that I think the documentation misses the details of how to configure the exact number of the cameras we are running. We are using two cameras and not quite sure about how the cancel the other two cameras in the control panel.

**Comment by g3force** [comment]
I have updated the wiki to reflect your questions:
https://github.com/RoboCup-SSL/ssl-vision/wiki/camera-calibration

Does this make it more clear?

**Comment by donglin1111** [comment]
seems like that the documentation is not quite specific on the field configuration details. In the SSL vision UI, all the parameters under field configuration have no units, the default values are 12000 for field length, 9000 for field width......and so on. I tried to find the units of these numbers but I just did not see it. Also I believe that we need to tell the SSL vision that how many cameras we are actually using before we do the field configuration, because currently we need to put the four bullet points (the four blue points) at the quad positions of the whole field to make the configuration looks like that it is correct......
